syntax = "proto3";
package plugin_version;





/*
UpdatePluginVersion请求
*/
message UpdatePluginVersionRequest {
    /*
    插件Id
    */
    string id = 1 ;
    /*
    版本Id
    */
    string verId = 2 ;
    /*
    版本名称
    */
    string name = 3 ;
    /*
    备注
    */
    string memo = 4 ;
}

/*
UpdatePluginVersion返回
*/
message UpdatePluginVersionResponse {
    /*
    ID
    */
    string id = 1 ;
}

/*
UpdatePluginVersionApi返回
*/
message UpdatePluginVersionResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    plugin_version.UpdatePluginVersionResponse data = 4 ;
}

